lib/cmd/fh3/credential/list.js

```diff
@@ -0,0 +1,84 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var _ = require('underscore');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('Operations on credential bundles'),
+  'examples' :
+  [{
+    cmd : 'fhc credential list',
+    desc : "List of all 'credential bundles' of this domain"
+  },{
+    cmd : 'fhc credential list --name=<name>',
+    desc : "List of all 'credential bundles' with the <name> of this domain"
+  },{
+    cmd : 'fhc credential list --platform=<platform>',
+    desc : "List of all 'credential bundles' with the <platform ios | Android> of this domain"
+  },{
+    cmd : 'fhc credential list --type=<type>',
+    desc : "List of all 'credential bundles' with the <type distribution | release | debug> of this domain"
+  },{
+    cmd : 'fhc credential list --json',
+    desc : "List of all 'credential bundles' of this domain with a json format"
+  }],
+  'demand' : [],
+  'alias' : {
+    'name': 'n',
+    'type': 'n',
+    'platform': 'p',
+    'json': 'j'
+  },
+  'describe' : {
+    'name' : i18n._("Name of the 'credential bundle'."),
+    'platform' : i18n._("Platform of the 'credential bundle' which can be 'ios' or 'Android'."),
+    'type' : i18n._("Type of the 'credential bundle' which can be 'distribution', 'release' or 'debug'."),
+    'json' : i18n._("Output in json format"),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)